src/v/cluster/controller_probe.cc

```diff
@@ -101,6 +102,25 @@ void controller_probe::setup_metrics() {
           sm::description(
             "Number of partitions that lack quorum among replicants"))
           .aggregate({sm::shard_label}),
+        sm::make_gauge(
+          "latest_cluster_metadata_manifest_age",
+          [this] {
+              auto maybe_manifest_ref
+                = _controller.metadata_uploader().manifest();
+              if (!maybe_manifest_ref.has_value()) {
+                  return int64_t{-1};
```

Comment:
I think 0 will be a more natural value to aggregate over.

Reply:
Let's also check if the manifest's timestamp is 0, and return 0 then too. It indicates we've never uploaded.